cypress/page/student-form.page.ts

```diff
@@ -0,0 +1,61 @@
+import { contains } from "cypress/types/jquery";
+
+class StudentFormPage {
+  private studentFormURL: string;
+  private studentForm: string;
+  
+  constructor() {
+    this.studentFormURL = "https://demoqa.com/automation-practice-form";
+    this.studentForm = "#userForm"
+  }
+
+  public visStudentFormPage () {
+    cy.visit(this.studentFormURL);
+  }
+
+  public fillForm(userInformation: {
+    name: any;
+    lastName: any;
+    email: string;
+    gender: string;
+    dateOfBirth: string;
+    mobileNumber: string;
+    hobbies: string[];
+    currentAddress: string;
+    })
```

Comment:
can you create an interface or type to represent the userInformation. See more information here: https://www.typescriptlang.org/docs/handbook/2/objects.html